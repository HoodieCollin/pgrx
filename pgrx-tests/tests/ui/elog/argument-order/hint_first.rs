use pgrx::prelude::*;

fn main() {}

#[pg_extern]
fn hint_first_is_invalid() {
    log!(
        hint = "hint";
        message = "...";
    );
}
