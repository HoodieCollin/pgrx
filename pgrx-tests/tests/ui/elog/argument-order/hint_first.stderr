error: The alternative invocation requires `message` to be specified first.
  --> tests/ui/elog/argument-order/hint_first.rs:7:5
   |
7  | /     log!(
8  | |         hint = "hint";
9  | |         message = "...";
10 | |     );
   | |_____^
   |
   = note: this error originates in the macro `log` (in Nightly builds, run with -Z macro-backtrace for more info)
